### PR TITLE
webview2 updates

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

File: docs/browser-engine/webview2.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+WebView2 is backed by Microsoft, and continues to engage with the developer community and take feature requests and bug fixes.
+
+## Release cadence, version support and update model
+
+<!-- ✍ How often is there a new version? Is it predictable? Does it use an evergreen model (i.e.:
+it gets updated automatically) or do developers need to take care of that? Does it depend on other
+projects to ship? -->
+
+There are 2 parts to WebView2: the SDK, which comes in the form of a NuGet package, and the WebView2 Runtime, which take and optimize the edge browser binaries. The SDK and Runtime get updated every 4 weeks, in alignment with Edge. WebView2 offers a prerelease SDK with up and coming experimental features and a stable release SDK.
```
Comment (molant) [inline docs/browser-engine/webview2.md:31]: I'd stick with the 6 release cadence and make [a note similar to the one in Electron](https://github.com/crossplatform-dev/crossplatform.dev/pull/6/files#diff-17c6f8d1d1ab64c44676e2301283cfcb698ac6ff80bed1defb5b9140c1395256R33) about that switching soon-ish.

Comment (molant) [reply docs/browser-engine/webview2.md:31]: Would it be possible to add links to the NuGet package and the Runtime?

Also how is the versioning supposed to work? Is there only one major version that gets updated every few weeks and supposedly no breaking changes? Are there minor updates between those 6 weeks periods?

We might want to move the distribution modes here (or mention them again) as it is part of the update process.

File: docs/browser-engine/webview2.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+projects to ship? -->
+
+There are 2 parts to WebView2: the SDK, which comes in the form of a NuGet package, and the WebView2 Runtime, which take and optimize the edge browser binaries. The SDK and Runtime get updated every 4 weeks, in alignment with Edge. WebView2 offers a prerelease SDK with up and coming experimental features and a stable release SDK. 
+
+## Governance
+
+<!-- ✍ Please add an introductory paragraph about the governance model of the project: part of a
+foundation, company driven, etc. -->
+
+WebView2 is backed by Microsoft, and is built on top of the Edge browser platform.
```
Comment (molant) [inline docs/browser-engine/webview2.md:38]: I don't think backed is appropriate as it is not OSS.

```suggestion
WebView2 is developed by Microsoft, and built on top of the Edge browser platform. 
```

What does `platform` mean here? Maybe it should be `built on top of Microsoft Edge`?

I'd also add that developers can report issues and propose features in the GitHub repo.

File: docs/browser-engine/webview2.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+## Summary
+
+| Characteristic |        |
+| -------------- | :----: |
+| Website | [{{technologies.webview2.url}}]({{technologies.webview2.url}}) |
+| Platforms | Desktop: Win32, WPF, WinForms |
+| Type | [Browser engine] |
+| Software type | close source |
+| License | https://www.nuget.org/packages/Microsoft.Web.WebView2/0.9.430/License |
+| Copyright | Micrsoft |
```
Comment (molant) [inline docs/browser-engine/webview2.md:49]:
```suggestion
| Copyright | Microsoft |
```

File: docs/browser-engine/webview2.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+WebView2 is backed by Microsoft, and is built on top of the Edge browser platform. 
+
+## Summary
+
+| Characteristic |        |
+| -------------- | :----: |
+| Website | [{{technologies.webview2.url}}]({{technologies.webview2.url}}) |
+| Platforms | Desktop: Win32, WPF, WinForms |
+| Type | [Browser engine] |
+| Software type | close source |
```
Comment (molant) [inline docs/browser-engine/webview2.md:47]:
```suggestion
| Software type | Clouse source |
```

File: docs/browser-engine/webview2.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+foundation, company driven, etc. -->
+
+WebView2 is backed by Microsoft, and is built on top of the Edge browser platform. 
+
+## Summary
+
+| Characteristic |        |
+| -------------- | :----: |
+| Website | [{{technologies.webview2.url}}]({{technologies.webview2.url}}) |
+| Platforms | Desktop: Win32, WPF, WinForms |
```
Comment (molant) [inline docs/browser-engine/webview2.md:45]: What do you think about skipping all the different type of Windows apps that can be done and just say the target is `Desktop`? I'd also add a note at the end saying that macOS is under development.

```suggestion
| Platforms | Desktop |
```

File: docs/browser-engine/webview2.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+| Type | [Browser engine] |
+| Software type | close source |
+| License | https://www.nuget.org/packages/Microsoft.Web.WebView2/0.9.430/License |
+| Copyright | Micrsoft |
+| Documentation | [{{technologies.webview2.documentation}}]({{technologies.webview2.documentation}}) |
+| Community | [{{technologies.webview2.community}}]({{technologies.webview2.community}}) |
+| Latest version | {{technologies.webview2.releases.0.version}} |
+| Release cadence | Major versions: 4 weeks <br/> Minor/patch: 4 weeks |
```
Comment (molant) [inline docs/browser-engine/webview2.md:53]: Let's stick with 6 weeks for now. Also, I thought there could be updates in between if needed. If that's the case maybe we could use `as needed`?

Also going back to my previous question about versioning, we should make sure this is aligned with that section.

File: docs/browser-engine/webview2.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+| Release support | 4 weeks |
```
Comment (molant) [inline docs/browser-engine/webview2.md:54]: This information should go in the Release cadence section and change it to 6 weeks.
Do you know if that's going to change with the 8 week support every other version that Chrome is planning?

File: docs/browser-engine/webview2.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+| Update model | Both Evergreen (automatic updates) or fixed model|
```
Comment (molant) [inline docs/browser-engine/webview2.md:55]:
```suggestion
| Update model | [Evergreen or fixed] |
```

And lets add a link to the documentation.

File: docs/browser-engine/webview2.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+| Governance model | WebView2 team + developer engagement |
```
Comment (molant) [inline docs/browser-engine/webview2.md:56]:
```suggestion
| Governance model | Microsoft driven |
```

File: docs/browser-engine/webview2.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+UWP support coming soon!
+:::
+
+**Language support:**
+
+{{ table technologies.electron.languages.{} }}
+
+<!-- Ref links -->
+
+[Browser engine]: ./browser-engine.md
```
Comment (molant) [inline docs/browser-engine/webview2.md:72]: Please add all the links through the document here so it is easier to update and read the markdown 🙏

**Review by molant — CHANGES_REQUESTED** [review]
Thanks a lot @jasonstephen15 🙏

Left a few suggestions. We can talk them over if there is something it is not clear or that you do not agree.
Also I see the `package-lock.json` file has been modified and I'm not sure why. Do you mind reverting that change?

File: docs/browser-engine/webview2.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+
+
+<!-- ✍ Please write about the technology, when it was created, what type of technology is it,
+some of the key characteristics for developers, etc. -->
+
+Althought the traditional IE WebBrowser control dates back to (2008 -- #JS can we find the actual date), WebView2 is a newer technology that has modernized the embeddable web browser by adopting the microsoft edge chromium platform. WebView2 has recently GA'd on Win32 C/C++, .NET, and expects to have UWP support soon. 
+
+WebView2 has both an evergreen and fixed distribution mode. With the default Evergreen mode, WebView2 will get the latest features, security, and bug-fixes automatically! For developers who want more control over their app updates we also offer the fixed version which allows you to ship webview2 with your app.
+
+WebView2 is backed by Microsoft, and continues to engage with the developer community and take feature requests and bug fixes.
```
[the remark this answers is not in the document]
Comment (jasonstephen15) [reply docs/browser-engine/webview2.md:23]: updated!

File: docs/browser-engine/webview2.md (hunk 7 above)
Comment (jasonstephen15) [reply docs/browser-engine/webview2.md:54]: It will stay with 4 weeks. We don't have plans for the extended stable atm.

**Comment by jasonstephen15** [comment]
@molant copied same package-lock as main

**Review by molant — APPROVED** [review]
Thanks a lot!

**Pull request merged**